manifests/elasticsearch.yaml

```diff
@@ -10,10 +10,12 @@ spec:
     protocol: TCP
     port: 9200
     targetPort: 9200
+    nodePort: 9200
```

Comment:
If we configure this on a service, will it mean it will route the request to any node of the cluster which can be different over time? (just curious)

Reply:
yes, this will open a port on every node, it will forward requests to a random ES